src/index.html

```diff
@@ -15,8 +15,95 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,400;1,500;1,700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <img
+        class="logo"
+        src="./images/logo.png"
+        alt="logo-moyo"
+      />
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item apple">
+            <a
+              class="apple is-active nav__link"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Smartphones
+            </a>
```

Comment:
Fix code style everywhere if you have only 2 attributes
```suggestion
            <a class="nav__link" href="#">
              Smartphones
            </a>
```

Reply:
Why is it happening after a "npm run lint" or "npm test"? Its makes this changes

Reply:
That's because of prettier config. Nevermind then